### ISSUE TITLE
WebRTC is not working from Docker while other protocols do

### ISSUE THREAD

**Issue by gehirndienst** [deepch/RTSPtoWeb#392]
I'm running the default docker container on Windows 11 with the following command:

`docker run -p 8083:8083 -d --name rtsp-to-web ghcr.io/deepch/rtsptoweb:latest`

I only have to replace the default config.json with my own because the default stream doesn't work at all. Then I saw on the webpage by clicking "all" that only webRTC is not working while other protocols do.  What could be the reason for that?

**Comment by bitraker** [comment]
I want to add I see a similar issue. I use the latest docker image, and have added a RTSP stream, from Unifi Protect. WebRTC which is the stream conversion I need for home assistant does not work, but MSE does.

**Comment by DcFamas** [comment]
Same problem here.

**Comment by DcFamas** [comment]
Thats true, sorry I thought the synology container is fully configured.
So for synology users; just change the network to host not bridge.
Thanks @allenporter !
